README.md

```diff
@@ -29,6 +29,12 @@ $ gem install debug --pre
 
 or specify `-Ipath/to/debug/lib` in `RUBYOPT` or each ruby command-line option, especially for debug this gem development.
 
+If you use Bunlder, write the following line to your Gemfile.
```

Comment:
Could you modify `misc/README.md.erb` instead of it? and `rake` (without taskname) will re-make `README.md`.

Reply:
Fixed it.